docs/content/how-to-guides/configure-reactions/configure-drasi-result-reaction/_index.md

```diff
@@ -3,8 +3,368 @@ type: "docs"
+curl -X GET "localhost:8080/query1"
+{{< /tab >}}
+
+{{< tab header="JavaScript" lang="JavaScript" >}}
+fetch("http://localhost:8080/query1", {
+  method: "GET"
+})
+  .then(response => {
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+    return response.json(); 
+  })
+  .then(data => {
+    // Output the data
+    console.log(data);
+
+    // Retrieve the header information
+    const headerObject = data.find(item => item.header);
+
+    const header = headerObject ? headerObject.header : null;
+
+    console.log(header);
+  })
+  .catch(error => {
+    console.error("Error:", error);
+  });
+
+{{< /tab >}}
+{{< /tabpane >}}
+
+### Retrieving the data from the current result set
+If you wish to retrieve only the data content of the result set, you can append `/data` to the end of your endpoint. 
+
+#### HTTP Request
+```bash
+GET http://localhost:<servicePort>/<queryId>/data
+```
+
+#### URL Parameters
+
+|Parameter|Description|
+|-|-|
+|servicePort|The port on your local machine that was forwarded earlier|
+|queryId|The id of the Continuous Query you are interested in|
+
+#### Returned Output
+The output from this GET Request will be an array of JSON elements.
+```bash
+[{"foo":"bar"},...]
+``` 
+
+#### Examples
+The examples below showcase how to retrieve the data of the current result set for a Continuous Query with the id of `query1`.
+
+{{< tabpane >}}
+{{< tab header="bash(curl)" lang="bash" >}}
+curl -X GET "localhost:8080/query1/data"
+{{< /tab >}}
+
+{{< tab header="JavaScript" lang="JavaScript" >}}
+fetch("http://localhost:8080/query1/data", {
+  method: "GET"
+})
+  .then(response => {
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+    return response.json(); 
+  })
+  .then(data => {
+    // Output the data
+    console.log(data);
+
+    // Retrieve the header information
+    const headerObject = data.find(item => item.header);
+
+    const header = headerObject ? headerObject.header : null;
+
+    console.log(header);
+  })
+  .catch(error => {
+    console.error("Error:", error);
+  });
+
+{{< /tab >}}
+{{< /tabpane >}}
+       
+
+### Retrieving the result set at a particular timestamp
+**NOTE:** This feature requires the `view` field of the Continuous Query to be set to `true`, and the `view/retentionPolicy` field to be set to `all` or `expire`. For more information, please navigate to this [link](/concepts/continuous-queries/#configuration).
+
+You can also use the Result Reaction to retrieve the result set for a specific timestamp, provided in milliseconds since the epoch.
+
+
+#### HTTP Request
+```bash
+GET http://localhost:<servicePort>/<queryId>/<timestamp>
+```
+
+#### URL Parameters
+
+|Parameter|Description|
+|-|-|
+|servicePort|The port on your local machine that was forwarded earlier|
+|queryId|The id of the Continuous Query you are interested in|
+|timestamp| The timestamp to retrieve the result set from|
```

Comment:
What is the format of the timestamp?

Reply:
It should be ms since epoch. I'll update the docs on this part